frontend/src/app/shared/tabs/tabs.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="tabs is-boxed">
+    <ul role="tablist" aria-label="select content to display">
+        <li *ngFor="let tab of tabs" [class.is-active]="activeTab === tab.id">
+            <a #tabLink id="{{tabLinkId(tab.id)}}" role="tab"
+                [attr.tabindex]="activeTab === tab.id ? 0 : -1"
```

Comment:
That means it is not reachable using the `TAB` key. (Perhaps somewhat confusing, tabindex refers to the tab key, it is not related to this being a tabs component).

Apparently the way to handle keyboard interaction is that you initially jump focus to the active tab, and use left/right keys to cycle through the links, rather than making each link part of the tabbing order.